src/data/faq_de.json

```diff
@@ -50,11 +120,70 @@
+                                "anchor": "ramp_down_reports",
+                                "active": false,
+                                "textblock": [
+                                    "Bis einschließlich 30. April.2023 können aus der App heraus noch Fehlerberichte an die Entwicklung übertragen werden. Danach ist es zwar möglich, eventuelle Fehler und Störungen bis einschließlich 31. Mai 2023 zu melden. Jedoch können Bugfixes zur noch für solche Fehler erstellt werden, die bis einschließlich 15. Mai 2023 gemeldet wurden. Entsprechend ist es auch in GitHub ab 16. Mai 2023 nicht mehr möglich, neue Issues zu eröffnen."
+                                ]
+                            },
+                            {
+                                "title": "Werden dann auch keine Zufalls-Kennungen für Begegnungen mehr ausgetauscht, wenn ich die App lösche?",
+                                "anchor": "ramp_down_exchange",
+                                "active": false,
+                                "textblock": [
+                                    "Der Austausch der Zufalls-Kennungen zur sog. Kontaktnachverfolgung (engl. Contact Tracing) erfolgt technisch durch das ENF-Modul (Exposure Notification Framework) Ihres Betriebssystems. Um sicherzustellen, dass nach einer Deinstallation der Corona-Warn-App auch das ENF-Modul deaktiviert ist, gehen Sie bitte wie folgt vor:",
+                                    "<details><summary>Schritt-für-Schritt Anleitung: iOS</summary><ul><li>Stellen Sie in den Einstellungen Ihres iPhones unter \"Begegnungsmitteilungen\" sicher, dass die Corona-Warn-App für die Region \"Deutschland\", und keine andere App, verwendet wird, sowie, dass die Begegnungsaufzeichnung noch aktiv ist. <div><img src='/assets/img/faq/ramp_down_exchange_iPhone_1_DE.png' width=300></div></li><li>Deinstallieren Sie die Corona-Warn-App von Ihrem iPhone</li><li>Sie sollten nun eine Benachrichtigung darüber erhalten haben, dass Geräte in der Nähe nicht mehr aufgezeichnet werden und Sie nicht mehr über mögliche Begegnungen informiert werden. <div><img src='/assets/img/faq/ramp_down_exchange_iPhone_2_DE.png' width=300></div></li><li>Kehren Sie zurück zu den \"Begegnungsmitteilungen\"-Einstellungen, indem Sie auf die zuvor erhaltene Benachrichtigung tippen. <div><img src='/assets/img/faq/ramp_down_exchange_iPhone_3_DE.png' width=300></div></li><li>Tippen Sie nun auf \"Begegnungsmitteilungen deaktivieren\"</li><li>Im unteren Bereich Ihres iPhones erscheint ein Hinweis, dass mit dem Deaktivieren des ENF-Moduls alle Daten und Mitteilungen gelöscht werden. Fahren Sie hierbei fort, indem Sie auf \"Deaktivieren und Daten löschen\" tippen. <div><img src='/assets/img/faq/ramp_down_exchange_iPhone_4_DE.png' width=300></div></li><li>Nun sind die Begegnungsmitteilungen auf Ihrem iPhone vollständig deaktiviert. <div><img src='/assets/img/faq/ramp_down_exchange_iPhone_5_DE.png' width=300></div></ul></details>",

```

Comment:
"Deaktivieren des ENF-Moduls"

also used here

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/73c82ad32454368aa9de62973567f834a1c8e0ca, thanks!